src/modules/report/types.ts

```diff
@@ -14,6 +14,8 @@ export type SendPdfReportRequestPayload = {
   now: string
   user: User
   applet: Applet
+  scoringType: string
+  subscaleTableData: Record<string, string>[] | null
```

Comment:
These properties will need to go on the `IActivityScoresAndReportsScores` type, as each score can have its own linked subscale

https://github.com/ChildMindInstitute/mindlogger-report-server/blob/b62a84f0827ead0250037c7be3caecee012a3873/src/core/interfaces/index.ts#L133-L142

Reply:
In fact, we should be sending the subscale itself because the list of items on the report with a linked subscale should come from the items on the subscale